corehq/apps/cloudcare/templates/cloudcare/config.html

```diff
@@ -138,17 +138,30 @@
 {% endblock %}
 
 {% block page_content %}
+    {% if request|is_new_cloudcare %}
+    <h2>{% trans 'Manage Web Apps Permissions' %}</h2>
+    {% else %}
     <h2>{% trans 'Manage CloudCare Permissions' %}</h2>
+    {% endif %}
     <section id="cloudcare-app-settings" style="display: none">
         <div data-bind="saveButton: saveButton"></div>
         <div id="cloudcare-app-settings-form" data-bind="with: applicationAccess">
             <label class="radio">
                 <input type="radio" value="false" data-bind="checked: restrict.JSON"/>
-                {% blocktrans %}Allow all mobile workers to see all CloudCare apps{% endblocktrans %}
+                {% blocktrans %}Allow all mobile workers to see all {% endblocktrans %}
+                {% if request|is_new_cloudcare %}
+                web apps.
```

Comment:
yeah i'm still unclear if web apps is a proper noun or just regular language. @amsagoff thoughts on whether it should be translated?